ports/windows/README.md

```diff
@@ -29,13 +32,37 @@ Install following packages using cygwin's setup.exe:
 
 Build using:
 
+    make
+or
     make CROSS_COMPILE=i686-w64-mingw32-
 
 Or for 64bit:
 
     make CROSS_COMPILE=x86_64-w64-mingw32-
 
 
+Building using MSYS2 / Mingw on Windows
+---------------------------------------
+
+Install MSYS2 64 bit from msys2.org
+Run the installer msys2-x86_64-xxxxxxxx.exe
+Follow the instructions on the msys2.org home page to update the package
+database and core modules.
+
+Install the compiler and make:
+
+    pacman -S mingw-w64-i686-gcc 
+    pacman -S make
+
+This will install gcc.exe which is identical to i686-w64-mingw32-gcc
+If you already have MSYS2 / Mingw installed run "gcc -dumpmachine"
+from a command prompt to ensure you have the correct gcc installed.
+Response should be "i686-w64-mingw32"
```

Comment:
Except when you're using the 64bit compiler it's ```x86_64-w64-mingw32```

Reply:
should be "i686-w64-mingw32" or "x86_64-w64-mingw32" if you have a 64 bit compiler installed. Either will work